src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -151,6 +155,25 @@ class GrabbitConfiguration {
     }
 
 
+    private static String schemeVal(Map<String, String> configMap, String key,
```

Comment:
I wonder if we should add some coverage around this in GrabbitConfigurationSpec?